lib/node_modules/@stdlib/stats/strided/ztest2/test/test.main.js

```diff
@@ -0,0 +1,342 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Float64Results = require( '@stdlib/stats/base/ztest/two-sample/results/float64' );
+var isfinite = require( '@stdlib/math/base/assert/is-finite' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var gfill = require( '@stdlib/blas/ext/base/gfill' ).ndarray;
+var normalFactory = require( '@stdlib/random/array/normal' ).factory;
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var NINF = require( '@stdlib/constants/float64/ninf' );
+var ztest2 = require( './../lib' );
+
+
+// VARIABLES //
+
+var normal = normalFactory({
+	'seed': 12345
+});
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof ztest2, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function performs a two-sample Z-test over a strided array (alternative=two-sided)', function test( t ) {
+	var results;
+	var out;
+	var x;
+	var y;
+
+	results = new Float64Results();
+
+	// Generate an array with a sufficiently large sample size to effectively guarantee expected results:
```

Comment:
```suggestion
	// Generate arrays with a sufficiently large sample size to effectively guarantee expected results:
```

You are generating multiple arrays. Applies here and elsewhere.